src/services/motion/client.test.ts

```diff
@@ -43,7 +43,6 @@ describe('moveOnGlobe', () => {
       type: 'component',
       subtype: 'base',
       name: 'myBase',
-      machinePartId: '',
```

Comment:
removing a reference to [a field that was removed in an unrelated proto update](https://github.com/viamrobotics/api/commit/274901dc0c4735cadb75792617c0700e8cde7bda)